src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/utils/RouterEnvelopesStatusChecker.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests.utils;
+
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+
+public class RouterEnvelopesStatusChecker {
+
+    private static final String blobRouterUrl = ConfigFactory.load().getString("blob-router-url");
+
+    /**
+     * Checks the status of envelope with given file name in blob-router.
+     */
+    public static String checkStatus(String fileName) {
+        var resp = RestAssured
```

Comment:
can't hurt to call it `response` now can it? :)

Reply:
Or maybe even use further objects which are used down here? `body()` or even `body().jsonPath()`?

Reply:
updated